README.md

```diff
@@ -33,6 +33,12 @@ docker run --rm -v /tmp/tbshare:/tbshare \
             --save-file /tbshare/status
 ```
 
+If you only want to monitor specific tables you can pass a whitelist flag like this:
+
+```bash
+    --whitelist schemaName1.tableName1 schemaName2.tableName2
```

Comment:
You forgot to change this to be csv also

Reply:
oh, thanks!